runtime/onert/backend/train/KernelGenerator.cc

```diff
@@ -324,7 +324,7 @@ void KernelGenerator::visit(const ir::train::operation::ElementwiseActivation &n
   };
 
   fn->configure(input_tensor, output_tensor, node.param().alpha, node.param().beta,
-                convertToInferActivationType(node.param().op_type));
+                node.param().approximate, convertToInferActivationType(node.param().op_type));
```

Comment:
I wondered why we need to update train code. 
After reading code more, I've found `train:ops:ElementwiseActivationLayer` inherits `cpu::ops::ElementwiseActivationlayer`.